### PR TITLE
Add EXCERPT parsing

### PR DESCRIPTION
**Pull request by otoyo**
Thank you for the very nice parser.
I added a new feature: `EXCERPT` parsing

### CODE REVIEW

File: parse_test.go [hunk 1]
```diff
@@ -7,7 +7,7 @@ import (
 	"testing"
 	"time"
 
-	. "github.com/catatsuy/movabletype"
+	. "github.com/otoyo/movabletype"
```
Comment (catatsuy) [inline parse_test.go]: Please don't change package name.

**Comment by catatsuy** [comment]
Sorry. I updated this repo code and released `v0.0.1`. If you change this PR, I will merge it and release it. @otoyo

File: parse_test.go (hunk 1 above)
Comment (otoyo) [reply parse_test.go]: Sorry, I fixed it at ae81c9a . Thank you.

**Review by catatsuy — APPROVED** [review]

**Comment by catatsuy** [comment]
Thank you! I released `v0.0.2`.

**Pull request merged**